src/test/run-make-fulldeps/use-suggestions-rust-2018/Makefile

```diff
@@ -0,0 +1,7 @@
+-include ../tools.mk
+
+all:
+	$(RUSTC) ep-nested-lib.rs
+
```

Comment:
dumb question but I forget -- in Makefiles -- what happens with a blank line like this? I think we should remove it :)

Reply:
afaik, nothing, and the other run-make tests do the same for separating the libs from the executables

Reply:
ok fine fine